### DISCUSSION TITLE
Create examples of indirect treatment comparison (ITC) using marginalisation

### DISCUSSION THREAD

**Discussion by n8thangreen** [HTAhackathon2024/.github#18] [Ideas]
### What is the problem we're trying to solve

Two contrasting approaching have recently been developed for doing ITC, which are better than MAIC or STC etc

- ML-NMR is a _conditional_ approach
- G-computation is a _marginal_ approach

ML-NMR has an excellent package called [multinma](https://cran.r-project.org/web/packages/multinma/index.html) which is mature and has lots of example vignettes.

The G-computation approach has a development package [here](https://github.com/StatisticsHealthEconomics/ModStanR) which doesn't have any examples. Can we repeat the `{multinma}` examples using G-computation?

### Which tools will we use?

- [ ] Review the `{multinma}` vignettes
- [ ] Write equivalent ones for G-computation, using the same data
- [ ] Include in G-computation package

### Previous work

These are the current vignettes:
https://dmphillippo.github.io/multinma/articles/example_atrial_fibrillation.html

**Comment by vikachooo** [comment]
I'd like to join this project!

**Reply by vikachooo** [reply]
Probably I'm more inclined towards the LLM in assertHE project .. :)
